### PR TITLE
Build with support for openblas

### PR DESCRIPTION
**Pull request by baldersheim**
@toregge PR

### CODE REVIEW

File: jllama-cuda/vespa-jllama-cuda.spec.tmpl [hunk 1]
```diff
@@ -35,6 +35,15 @@ BuildRequires: cuda-compiler-12-2
 BuildRequires: cuda-cudart-devel-12-2
 BuildRequires: cuda-command-line-tools-12-2
 %endif
+
+%if 0%{?el8} || 0%{?el9}
```
Comment (toregge) [inline jllama-cuda/vespa-jllama-cuda.spec.tmpl:39]: Either drop el9 here or make vespa use vespa-openblas for el9.

File: jllama/vespa-jllama.spec.tmpl [hunk 2]
```diff
@@ -48,6 +48,14 @@ BuildRequires: vespa-cmake
 BuildRequires: make
 BuildRequires: git
 
+%if 0%{?el8} || 0%{?el9}
```
Comment (toregge) [inline jllama/vespa-jllama.spec.tmpl:51]: Either drop el9 here or make vespa use vespa-openblas for el9.

File: jllama/vespa-jllama.spec.tmpl [hunk 3]
```diff
@@ -48,6 +48,14 @@ BuildRequires: vespa-cmake
...
+BuildRequires: vespa-openblas-devel >= 0.3.27
+Requires: vespa-openblas >= 0.3.27
+%else
+BuildRequires: openblas-devel
+Requires: openblas
```
Comment (toregge) [inline jllama/vespa-jllama.spec.tmpl:56]: vespa-base-libs requires openblas-serial when el8 is not defined and vespa-openblas when el8 is defined. Is mixing of different openblas libraries intentional ?

File: jllama-cuda/vespa-jllama-cuda.spec.tmpl [hunk 4]
```diff
@@ -35,6 +35,15 @@ BuildRequires: cuda-compiler-12-2
...
+BuildRequires: vespa-openblas-devel >= 0.3.27
+Requires: vespa-openblas >= 0.3.27
+%else
+BuildRequires: openblas-devel
+Requires: openblas
```
Comment (toregge) [inline jllama-cuda/vespa-jllama-cuda.spec.tmpl:44]: vespa-base-libs requires openblas-serial when el8 is not defined and vespa-openblas when el8 is defined. Is mixing of different openblas libraries intentional ?

**Comment by toregge** [comment]
Note that you get an auto require on `libopenblas.so.0()(64bit)` which is _not_ auto provided by `vespa-openblas`.

File: jllama-cuda/vespa-jllama-cuda.spec.tmpl (hunk 1 above)
Comment (baldersheim) [reply jllama-cuda/vespa-jllama-cuda.spec.tmpl:39]: hmm, I assumed we used it for both as it is built for both. Lets discuss tomorrow.

**Review by toregge — APPROVED** [review]

**Pull request merged**